README.md

```diff
@@ -501,9 +501,16 @@ Example:
 
 Yes.
 
-### Does this library work with Angular 5.x?
+### Does this library work with Angular Versions < 8.0.0?
 
-The latest version that supports Angular 5.x is 1.1.1. You can still install it via npm or yarn, e.g. `npm install ngx-drag-to-select@1.1.1`.
+Please have a look here to see which version supports your angular application. 
```

Comment:
I'd re-write this to something like (given we use the title I suggested):

```
Yep you totally can! But this also means you won't be able to use the latest and greatest and all of the newer features. The reason for this is that we are not back porting new features to older versions of this library, due to maintenance overhead. Severe bugs and security issues, however, are back ported.

What does that mean for you now? We recommend to stay up to date with new Angular versions. If for some reasons you can't then here's an overview versions you could use with older versions of Angular:

<TABLE>

To install a specific version run for example `npm install ngx-drag-to-select@x.x.x`.
```

And then we can also get rid of the sentence below the table.